backend/src/appointment/l10n/en/email.ftl

```diff
@@ -10,6 +10,8 @@
                 {-brand-name}
                 {-brand-slogan} {-brand-sign-up-with-url}
 
+mail-brand-contact-form = contact form
+mail-brand-support-hint = Got questions? We're here to help. Fill out our { $contact_form_link }, or simply reply to this email for support.
```

Comment:
We need a reply-to header on the email first :eyes:

Reply:
Whoops of course 😅

Reply:
We already have that env: `SUPPORT_EMAIL` I'll use that for the reply-to header